doc/integrator/admin_pages.rst

```diff
@@ -0,0 +1,61 @@
+.. _integrator_admin_pages:
```

Comment:
We might rename this file `admin_interface.rst`

so we get here:

.. _integrator_admin_interface:

And the section tile could be "Configure the admin interface"